README.md

```diff
@@ -31,19 +31,10 @@ yarn
 yarn test
 ```
 
-Build and watch Gestalt:
+Build and watch Gestalt & run the docs server:
 
 ```bash
-cd packages/gestalt
-yarn watch
-```
-
-Start the docs server:
-
-```bash
-cd docs
 yarn start
-open "http://localhost:8080"
```

Comment:
Before the default was `3000`, however this didn't allow us to run it concurrently with the masonry playground (see below)

We also don't need to run the `open` command, create-react-app automatically does that for us.